vetiver/server.py

```diff
@@ -43,6 +43,13 @@ class VetiverAPI:
 
     Notes
     -----
+    This generates an API with either 2 or 3 GET endpoints and 1 POST endpoint.
+
+    ├──/ping (GET)
+    ├──/metadata (GET)
+    ├──/pin-url (GET, if model is pinned on Posit Connect)
```

Comment:
Does the URL not get added to the API if it is pinned elsewhere, like on S3 or Azure or similar?